packages/packages/top-bar/src/components/ui/popover-menu.tsx

```diff
@@ -0,0 +1,34 @@
+import { Menu, styled, MenuProps, paperClasses, menuClasses } from '@elementor/ui';
+import { MenuContextProvider } from '../../contexts/menu-context';
+
+type ExtraProps = {
+	spacing?: string | number;
+}
+
+const StyledMenu = styled( ( props: MenuProps ) => (
+	<Menu elevation={ 0 } { ...props } />
+), {
+	shouldForwardProp: ( prop ) => 'spacing' !== prop,
+} )<ExtraProps>( ( { spacing = '13px' } ) => ( {
+	[ `& .${ paperClasses.root }` ]: {
+		borderRadius: '6px',
+		minWidth: '200px',
+		color: 'white',
+		backgroundColor: '#232629',
+		marginTop: spacing,
```

Comment:
all these values should be read from the theme

Reply:
Need to be in another task